src/crush/CrushCompiler.cc

```diff
@@ -815,6 +815,44 @@ void CrushCompiler::dump(iter_t const& i, int ind)
     dump(i->children.begin() + j, ind+1); 
 }
 
+int CrushCompiler::adjust_bucket_item_place(iter_t const &i)
```

Comment:
@tchaikov  Thank you sir for review! I will add some comments about this function.And the function name use "change" or "move" take place the "adjust"?